server/routes/ads.js

```diff
@@ -1,25 +1,12 @@
 const express = require('express');
 const router = express.Router();
 const { google } = require('googleapis');
-const jwtClient = new google.auth.JWT(
-  process.env.CLIENT_EMAIL,
-  null,
-  process.env.PRIVATE_KEY,
-  ['https://www.googleapis.com/auth/spreadsheets']
-);
-
-//authenticate request
-jwtClient.authorize(function (err, tokens) {
-  if (err) {
-    console.log(err);
-  } else {
-    console.log('Successfully connected to Google Spreadsheet!');
-  }
-});
-
+const jwtClient = require('../bin/jwtClient.js');
 const spreadsheetId = process.env.SPREADSHEET_ID;
 const sheets = google.sheets('v4');
-let sheetRange = 'Ads!A2:J10';
+const uuidv4 = require('uuid').v4;
+const moment = require('moment');
+let sheetRange = 'Ads!A2:J';
```

Comment:
will this always insert in the last row?

Reply:
Yeah, far as I know. Screenshot from the docs
![image](https://user-images.githubusercontent.com/4298117/115088393-ed7b6e00-9ec4-11eb-9c3b-110d02d66103.png)